script.js

```diff
@@ -180,7 +180,7 @@ const renderCards = (indx) => {
     if (cardIndx === indx) {
       document.querySelector('.pop-up').innerHTML = `
       <div class="mult">
-      <div class="ham">
+      <div class="ham va">
```

Comment:
## As required, Kindly ensure that the data is preserved in the browser upon reloads. Ensure:
1. After filling the form, and not submitting it, the entered values should be saved in the browser.
2. When visiting the page the form inputs should be contained, be auto-populated with, the values entered in the previous visit to the page.
3. If at the previous visit to the page, in the same browser, no input was entered in the form then the form inputs should remain empty.

- [ ] As required, Kindly save all input data in a single object, saved to a single local storage registry.

## EXAMPLE
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```

- [ ] As required, I kindly suggest that you make use of `localStorage.getItem` and `localStorage.setItem` functions to save and retrieve the form data object to the local storage.

## EXAMPLE
`localStorage.setItem('contactFormInput', JSON.stringify(formData));`


- [ ] Kindly ensure that when the user loads the page, the code checks if there's any saved data in the local storage and if so the input fields are pre-filled with the data.